### DISCUSSION TITLE
Inactive users should be unranked

### DISCUSSION THREAD

**Discussion by Jamiras** [RetroAchievements/RAWeb#728] [Ideas]
As a user, I shouldn't have to compete with inactive users. 

We have a special "untracked" user status that prevents users from appearing in the rankings. This is a request to add an "inactive" flag to users that haven't logged in recently (within the last six months?). Whatever logic is filtering out the "untracked" users should also filter out the "inactive" users. We currently have ~61000 accounts, but likely less than 20% of those have been used in the last six months.

Additionally, users without any points should be ignored when calculating "Top N%". Having just 1 point is enough to get you in the top 67% (see http://retroachievements.org/User/Norwalk)

**Comment by kdecks** [comment]
I like this. Secondly, anyone without points shouldn't be ranked at all.

**Comment by SyrianBallaS** [comment]
I think this is a good idea, it might affect me as well since I'm almost never playing using RA anymore but it still sounds good.

**Comment by Jamiras** [comment]
#580 limits ranked users to a minimum of 500 points (250 now with the softcore split).

It's probably less important to filter out inactive users now that there's a minimum threshold as that filters out most of the users who tried the site and left. I'm not sure how much of the userbase above the threshold is inactive.

**Comment by WinteringGoose** [comment]
I don't know if this is still being considered, but especially now that there's a threshold to be ranked anyway, I think it's a bad idea. To state my serious point in a silly way, Ash wants to be the best there *ever was,* not just the best there *is.*
